### PR TITLE
Fix CI by using Github Actions from Haskell Organization

### PR DESCRIPTION
**Pull request by turion**

### CODE REVIEW

**Comment by turion** [comment]
I don't understand what went wrong here

**Comment by miguel-negrao** [comment]
That is a very strange error, it thinks the package is its own dependency ? hum...

**Comment by miguel-negrao** [comment]
I believe ImportQualifiedPost is only available from 8.10 up...

File: .github/workflows/haskell.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@ jobs:
 
     strategy:
       matrix:
-        ghc: ['8.8.4','8.10.7','9.0.2']
+        ghc: ['8.8','8.10','9.0','9.2']
```
Comment (turion) [inline .github/workflows/haskell.yml]:
```suggestion
        ghc: ['8.10','9.0','9.2']
```

File: .github/workflows/haskell.yml [hunk 2]
```diff
@@ -16,11 +16,11 @@ jobs:
 
     strategy:
       matrix:
-        ghc: ['8.8.4','8.10.7','9.0.2']
+        ghc: ['8.10','9.0','9.2']
```
Comment (turion) [inline .github/workflows/haskell.yml]:
```suggestion
        ghc: ['8.10']
```

**Pull request merged**